lib/Dialect/Comb/CombFolds.cpp

```diff
@@ -136,6 +136,86 @@ OpFoldResult ExtractOp::fold(ArrayRef<Attribute> constants) {
   return {};
 }
 
+static LogicalResult extractConcatToConcatExtract(ExtractOp op,
```

Comment:
Plz add a doc comment (similar to the one you have on the "if" that calls this below) describing the transformation.  It is a violation of DRY but helps when scanning the functions in the code.